config/routes.rb

```diff
@@ -19,9 +19,9 @@
 
   scope :cart do
     get "/", to: "orders#cart", as: "cart"
-    get "checkout", to: "orders#checkout"
-    patch "checkout", to: "orders#update"
-    get "receipt", to: "orders#receipt"
+    get "/checkout", to: "orders#checkout", as: "checkout"
+    patch "/checkout", to: "orders#update"
+    get "/receipt", to: "orders#receipt", as: "receipt"
```

Comment:
I just figured we should be consistent, and since we did it for login/logout, it made sense to me.